tests/lib/rules/jsx-no-target-blank.js

```diff
@@ -197,6 +197,11 @@ ruleTester.run('jsx-no-target-blank', rule, {
       output: '<a target="_blank" rel="no referrer noreferrer" href="https://example.com/7"></a>',
       errors: defaultErrors,
     },
+    {
+      code: '<a href="https://example.com/7a-run-on" target="_blank rel=noopener"></a>',
```

Comment:
I think i could see a general rule for "on an html element, is there an attribute value that has an `=` sign in it where the LHS of the `=` sign is a valid attribute" - which would cover this case. that would share some knowledge with the `no-unknown-property` rule, and would likely be able to offer suggestions but NOT autofix. Is that something you're interested in building?